### PR TITLE
Atom Subtabs

### PR DESCRIPTION
**Pull request by mboperator**
## Changes
- Added Subtabs
- Pretty BEM heavy -- CR appreciated!

## Screenshots
screen shot 2017-09-27 at 9 02 50 am

## Design Spec
https://procore.invisionapp.com/d/main#/console/12028911/253477089/inspect

### CODE REVIEW

**Comment by remyyounes** [comment]
As a follow up, let's remove all the inline styles in the index file.

File: src/styles/generic/_list.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@import "../settings/variables";
+
+ol, ul {
+  list-style-position: inside;
+}
+
+.list {
+  margin: 5px 0px;
```
Comment (remyyounes) [inline src/styles/generic/_list.scss]: move to margin variables

File: src/styles/generic/_list.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  &__item {
+    font-size: 12px;
+    font-weight: 400;
+    height: 28px;
+    list-style-type: none;
+    padding: 5px 15px;
```
Comment (remyyounes) [inline src/styles/generic/_list.scss]: move to padding variables

File: src/styles/generic/_list.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ol, ul {
+  list-style-position: inside;
+}
+
+.list {
+  margin: 5px 0px;
+
+  &__item {
+    font-size: 12px;
+    font-weight: 400;
```
Comment (remyyounes) [inline src/styles/generic/_list.scss]: follow up: use a font class (up-coming PR)

File: src/styles/components/_tabs.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      cursor: default;
+      font-weight: 900;
+      border-bottom: 2px solid palette(orange);
+    }
+
+    &--dropdown {
+      position: relative;
+
+      &::after {
+        content: url("data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iOHB4IiBoZWlnaHQ9IjhweCIgdmlld0JveD0iMCAwIDggOCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj4KICAgIDwhLS0gR2VuZXJhdG9yOiBTa2V0Y2ggNDYuMiAoNDQ0OTYpIC0gaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoIC0tPgogICAgPHRpdGxlPkNhcmV0PC90aXRsZT4KICAgIDxkZXNjPkNyZWF0ZWQgd2l0aCBTa2V0Y2guPC9kZXNjPgogICAgPGRlZnM+PC9kZWZzPgogICAgPGcgaWQ9IlBhZ2UtMS1Db3B5IiBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4KICAgICAgICA8ZyBpZD0iQ2FyZXQiIGZpbGw9IiM0RDRENEQiPgogICAgICAgICAgICA8cG9seWdvbiBpZD0iUmVjdGFuZ2xlLTEyIiBwb2ludHM9IjAgMiA4IDIgNCA2Ij48L3BvbHlnb24+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4=");
```
Comment (remyyounes) [inline src/styles/components/_tabs.scss]: did you mean to use base64 here ?

File: src/styles/generic/_list.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+@import "../settings/variables";
+
+ol, ul {
+  list-style-position: inside;
+}
+
+.list {
```
Comment (justinmwatts) [inline src/styles/generic/_list.scss]: Not sure if we decided to continue to follow ITCSS for css organization but if we wanted to continue with ITCSS, .list would go in a /components file. Element styling on lines 3-5 would go in a /elements file. A good read on the ITCSS layers: https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/

Comment (mboperator) [reply src/styles/generic/_list.scss]: Thanks @justinmwatts ! This diagram is incredibly helpful.

**Pull request merged**